config/getter.go

```diff
@@ -183,6 +175,14 @@ func GetTokenIssuer() string {
 	return viper.GetString("auth.jwt_issuer")
 }
 
+func GetGoogleClientId() string {
```

Comment:
Can you add these to heroku as well otherwise we might forget

Reply:
https://app.zenhub.com/workspaces/the-shoe-project-f20-5f729b8f69fd6b2517b9a848/issues/uwblueprint/shoe-project/243